File: data/PersonObject/Person.cs
using System;

public class Person
{
    // Private fields
    private string name;
    private int age;

    // Constructor
    public Person(string name, int age)
    {
        this.name = name;
        this.age = age;
    }

    // Getter for name
    public string GetName()
    {
        return name;
    }

    // Setter for name
    public void SetName(string name)
    {
        this.name = name;
    }

    // Getter for age
    public int GetAge()
    {
        return age;
    }

    // Setter for age
    public void SetAge(int age)
    {
        this.age = age;
    }

    // Method to display person details
    public void DisplayDetails()
    {
        Console.WriteLine($"Name: {name}, Age: {age}");
    }

    // Main method for testing
    public static void Main(string[] args)
    {
        Person person = new Person("John Doe", 30);
        person.DisplayDetails();
    }
}

File: data/PersonObject/Person.py
class Person:
    # Constructor
    def __init__(self, name, age):
        self.name = name
        self.age = age

    # Getter for name
    def get_name(self):
        return self.name

    # Setter for name
    def set_name(self, name):
        self.name = name

    # Getter for age
    def get_age(self):
        return self.age

    # Setter for age
    def set_age(self, age):
        self.age = age

    # Method to display person details
    def display_details(self):
        print(f"Name: {self.name}, Age: {self.age}")

# Testing the class
if __name__ == "__main__":
    person = Person("John Doe", 30)
    person.display_details()